python_modules/libraries/dagster-components/dagster_components_tests/code_locations/python_script_location/components/scripts/component.yaml

```diff
@@ -5,7 +5,13 @@ params:
     - path: script_one.py
       assets:
         - key: a
+          automation_condition:
+            type: eager
         - key: b
+          automation_condition:
+            type: on_cron
```

Comment:
Can we figure out a way to make it so we can do

`automation_condition: "@daily"` for simple cases?

Reply:
Per offline discussion we decided to go forward with this highly generic format and see how it feels.